h2o-core/src/main/java/water/H2O.java

```diff
@@ -615,11 +625,15 @@ else if(s.matches(("client_disconnect_timeout"))){
           throw new IllegalArgumentException("Interval for checking if client is disconnected has to be positive (milliseconds).");
         }
         trgt.clientDisconnectTimeout = clientDisconnectTimeout;
-        }
-      else if(s.matches("useUDP")) {
-          Log.warn("Support for UDP communication was removed from H2O, using TCP.");
-      }
-      else {
+      } else if (s.matches("useUDP")) {
+        Log.warn("Support for UDP communication was removed from H2O, using TCP.");
+      } else if (s.matches("disable_beta")) {
+        Log.info("Disabling algorithms marked as BETA.");
```

Comment:
I would also disable experimental here - I liked your idea of "levels" - I would just not surface the levels to the user and leave them just a simple option without parameters.

-disable_beta disables beta and experimental,
-disable_experimental disables only experimental algos.

If you rather want to have the original option that takes a parameter, that is fine as well. I didn't want to make it more complicated - but my suggestion might be actually worse than what you originally had. WDYT?

Reply:
Levels are intuitive and make sense. If one doesn't want to use beta, purely experimental algos are probably out of game too.

Can't say there is not a single user who would use experimental and not beta.

Full control would require disabling algos selectively, which seems to be way too much obviously. I'd keep the levels for simplicity and we can adjust later. Our algos are categorized well and there is no need to fear. Curently, this only touches `IRF` anyway. So there is time to adjust.

**Reverting back to single-flag with explicit options.**